layouts/shortcodes/ct_logs.html

```diff
@@ -1,8 +1,23 @@
-{{ range $.Site.Data.transparency.transparencyLogs }}
-<p>
-  <strong>{{ i18n "certificate_transparency_name" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .name }}</strong><br>
-  {{ i18n "certificate_transparency_role" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ if eq .role "production" }}{{ i18n "certificate_transparency_role_production" }}{{ end }}{{ if eq .role "test" }}{{ i18n "certificate_transparency_role_test" }}{{ end }}<br>
-  {{ i18n "certificate_transparency_uri" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .uri }}<br>
-  {{ i18n "certificate_transparency_public_key" }}{{ .Site.Params.beforeColon | safeHTML }}: <code>{{ .publicKey }}</code>
-</p>
+{{ $inner := .Inner }}
+{{ $data := .Get "data" }}
+{{ range $.Site.Data.transparency.categories }}
+    {{ $category := . }}
+    {{ if (eq $category $data "$category") }}
+    {{ $i18nVar := (printf "%s%s" "certificate_transparency_role_" $data) }}
+        <h3>{{ i18n $i18nVar }}</h3>
+        {{ $inner | markdownify }}
+
+        <ul>
+        {{ range $.Site.Data.transparency.transparencyLogs }}
+            {{ if and .role (eq $category .role) }}
+            <li>
+            <strong>{{ i18n "certificate_transparency_name" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .log | humanize }} {{ .shard }}</strong><br>
+            {{ i18n "certificate_transparency_uri" }}{{ .Site.Params.beforeColon | safeHTML }}: https://{{ .log }}.ct.letsencrypt.org/{{ .shard }}<br>
+            {{ i18n "certificate_transparency_public_key" }}{{ .Site.Params.beforeColon | safeHTML }}: <code>{{ .publicKey }}</code>
+            </li>
+            {{ end }}
+        {{ end }}
+    </ul>
+    </ul>
```

Comment:
Why this `ul` is opened in the markdown instead of here between line 7 and 8, just before {{ $inner }} ? Currently both this template and the markdown looks buggy even if it's not the case.

Reply:
This is due to bulleted list grouping for screen reader accessibility. When the partial (shortcode?) is merged with the document, it will properly nest the ct logs as sub-bullets instead of two separate bulleted lists. I

Reply:
Yes, I understand, but in your current code, you wrote:
```html
{{< ct_logs data="production" >}}
<ul>
    <li>...</li>
{{< /ct_logs >}}

{{< ct_logs data="testing" >}}
<ul>
    <li>...</li>
{{< /ct_logs >}}
```
So, because in **both** calls you have the `<ul>` why not doing that : 

```html
{{< ct_logs data="production" >}}
    <li>...</li>
{{< /ct_logs >}}

{{< ct_logs data="testing" >}}
    <li>...</li>
{{< /ct_logs >}}
```

and add the `<ul>` there in the shortcode just before `inner`: 

```html
        <h3>{{ i18n $i18nVar }}</h3>
<ul>
{{ $inner | markdownify }}
```
The resulting html will be the same, or am I missing something? (and, I think, the code more readable)

Reply:
Ah I gotcha, let me try that out.

Reply:
The html renders incorrectly using that method. Check this picture out.
![Screenshot_2020-02-06_11-36-03](https://user-images.githubusercontent.com/2382565/73957837-fe018300-48d4-11ea-8ccb-a67c887f232a.png)

Reply:
Try without `| markdownify ` : the inner var already contains html, no need to transform markdown into html.

Reply:
That did the trick :D